GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -93,8 +93,8 @@ extension AssetsCatalogParser {
 // MARK: - ACTool
 
 extension AssetsCatalogParser {
-  fileprivate func loadAssetCatalog(at path: String) -> [[String: Any]]? {
-    let command = Command("xcrun", arguments: "actool", "--print-contents", path)
+  fileprivate func loadAssetCatalog(at path: Path) -> [[String: Any]]? {
+    let command = Command("xcrun", arguments: "actool", "--print-contents", String(describing: path))
```

Comment:
Doesn't PathKit's `Path` have a dedicated property to get the path string?
I mean, `String(describing:)` feels more intended for debug representation purposes to me, while something like `path.path` or `path.string` if it exists would feel more right (a bit like you'd use `url.absoluteString`)